versioned_docs/version-8.7/self-managed/tasklist-deployment/tasklist-configuration.md

```diff
@@ -131,6 +131,34 @@ Due to a known [bug](https://github.com/camunda/camunda/issues/31238), changes t
 
 :::
 
+### Settings for index templates priority
+
+Tasklist creates index templates that Elasticsearch uses for the historical indices. The priority of these templates can be changed.
+
+This is useful when the Elasticsearch provider has some predefined wildcard (with `*` index pattern) index templates with given priority, setting a higher priority for Tasklist index templates ensures that the correct index mappings and settings are applied on the indices created from these templates.
+
+The following configuration parameter defines the setting:
+
+| Name                                                 | Description                                          | Default value   |
+| ---------------------------------------------------- | ---------------------------------------------------- | --------------- |
+| camunda.tasklist.elasticsearch.indexTemplatePriority | Priority for all index templates created by Tasklist | - (no priority) |
+
+For OpenSearch:
+
+| Name                                              | Description                                          | Default value   |
+| ------------------------------------------------- | ---------------------------------------------------- | --------------- |
+| camunda.tasklist.opensearch.indexTemplatePriority | Priority for all index templates created by Tasklist | - (no priority) |
```

Comment:
❓ Why did you merge the ES/OS one for tasklist but not Operate? I like the merged version better

Reply:
Yeah fair enough. Tagging @christinaausley as an FYI as it somewhat relates to the meeting we had earlier about 8.8 configuration documentation

Reply:
I can merge OS/ES together into one unified doc that pulls both the Tasklist and Operate content in uniformly -- feel free to merge this and I can take it from here.